crawling/koreatech_bus/cheonan_shuttle.yaml

```
@@ -116,6 +116,14 @@ from_school:
       - node_name: *koreatech
         arrival_time: "22:00"
 
+  - route_name: "주중(22시 10분)"
+    running_days: [ "MON", "TUE", "WED", "THU", "FRI" ]
+    arrival_info:
+      - node_name: *koreatech
+        arrival_time: "22:10"
+      - node_name: *terminal
+        arrival_time: "22:35"
```

```suggestion
  - route_name: "주중(22시 10분)"
    running_days: [ "MON", "TUE", "WED", "THU", "FRI" ]
    arrival_info:
      - node_name: *koreatech
        arrival_time: "22:10"
      - node_name: *terminal
        arrival_time: "22:35"
      - node_name: *cheonan_station
        arrival_time: "종점"
```
천안역까지 운행한다는 점을 추가해주어야 할 것 같습니다!
다른 부분에서 도착시각을 "승하차"로 설정한 부분이 있어서 여기서는 "종점"으로 작성하는 것도 좋아보입니다.
다만 해당 부분이 정상적으로 동작하는지 스테이지에서 **반드시** 확인해봐야 할 것 같아요.

---

넵 알겠습니다~!!